### ISSUE TITLE
type of text encoder

### ISSUE THREAD

**Issue by dt-yuhui** [qiaoyu-zheng/RP3D-Diag#7]
Hi,
I found the text encoder in your paper and hf repo is "ncbi/MedCPT-Query-Encoder". 
However, in your github repo it is "FremyCompany/BioLORD-2023"
So which one did you finally choose?

**Comment by dt-yuhui** [comment]
Also, i'm interested in how to pretrain text encoder with knowledge enhancement.
Will this code be made public?

**Comment by qiaoyu-zheng** [comment]
Thanks for your interest. In fact, they are similar. The text encoder in our first version is medcpt, which is finetuned using our data. However, later we find that biolord is also a good choice which needs no more finetuning. You can directly use biolord aligned with our code. I think its a easier way for reproducing.

**Comment by dt-yuhui** [comment]
Thanks for your quick reply!

So there's **no** knowledge enhancement in the model if we choose "biolord" as text encoder?
But i'm still quite interested in how to inject domain-specific knowledge into pretrained text encoder...like how to mix synonyms, explanation and which loss is used in the ICD tree?

I couldn't find the relevant code in this repo. If possible, could you send this part of the code to my email: taoyh24@m.fudan.edu.cn.
Thanks again :)
